dev/tools/gen_keycodes/data/keyboard_maps.tmpl

```diff
@@ -40,6 +40,7 @@ const Map<int, PhysicalKeyboardKey> kFuchsiaToPhysicalKey = <int, PhysicalKeyboa
 };
 
 /// Maps macOS-specific key code values representing [PhysicalKeyboardKey].
+/// MacOS doesn't provide a scan code, but a virtual keycode to represent a physical key.
```

Comment:
should there be a blank line between the two lines?

Reply:
Yes.  I updated it in a separate PR: https://github.com/flutter/flutter/pull/30938